doc/users/prev_whats_new/github_stats_3.3.0.rst

```diff
@@ -254,7 +254,7 @@ Pull Requests (1066):
 * :ghpull:`17617`: Rewrite pdf test to use check_figures_equal.
 * :ghpull:`17654`: Small fixes to recent What's New
 * :ghpull:`17649`: MNT: make _setattr_cm more forgiving
-* :ghpull:`17644`: Doc 33 whats new consolidation
+* :ghpull:`17644`: Doc 33 what's new consolidation
```

Comment:
[As mentioned before](https://github.com/matplotlib/matplotlib/pull/28579#discussion_r1680888189) I would like to keep all PR titles exactly as they are, including any spell errors.

Reply:
I already went and fixed the PR titles, so these are fine, if a bit overzealous. Only the "tolerence" one should stay as it was as it is intentional.